systemtest/src/main/java/io/strimzi/systemtest/Resources.java

```diff
@@ -750,7 +750,7 @@ private Deployment getDeploymentFromYaml(String yamlPath) {
     }
 
     public DoneableDeployment clusterOperator(String namespace) {
-        return clusterOperator(namespace, "300000");
+        return clusterOperator(namespace, Long.toString(Constants.CO_OPERATION_TIMEOUT_DEFAULT));
```

Comment:
Why is the API of `clusterOperator(String, *String*)` and not `clusterOperator(String, long)`?

Reply:
Cause it's set as env variable of type `EnvVar` which allow set only Strings. In general I can do conversion right before it's set into `EnvVar` instead of on all other places. Guess it's better. solution.